.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+branch = True
+source = numba_dppy
+omit = numba_dppy/tests/*, numba_dppy/_version.py, *__init__.py
```

Comment:
Why `*__init__.py` omitted?

Reply:
__ init__.py files must be in the report. Fixed.